dreamhome.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from database.db import initialize_db
+
+app = Flask(__name__)
+
+app.config['MONGODB_SETTINGS'] = {
+    'host': 'mongodb://localhost/dreamhome'
+}
+
+initialize_db(app)
+
+@app.route("/")
```

Comment:
Eventually, we will probably move this information to the welcome/documentation controller